tests/shutdown/svirt_upload_assets.pm

```diff
@@ -38,6 +39,8 @@ sub extract_assets {
 }
 
 sub run {
+    # Skip if the os-autoinst backend implements this feature
+    return 1 if defined &backend::svirt::do_extract_assets;
```

Comment:
how about checking for the API version number from os-autoinst OpenQA/Isotovideo/Interface.pm